File: Problems/StopWatch_Sample/CS/StopWatch.cs
using System;
using System.Collections.Generic;
using System.Linq;

public class Program
{
    public void Main()
    {
        System.Diagnostics.Stopwatch sw = new System.Diagnostics.Stopwatch();

        Console.WriteLine("Hit Return Key...");
        sw.Start();

        Console.ReadLine();
        sw.Stop();

        Console.WriteLine("Execute time ... " + sw.ElapsedMilliseconds.ToString() + "ms\n");
    }
}

File: Problems/StopWatch_Sample/Python3/StopWatch.py
# coding: utf-8

import time


def main():
    time0 = time.time()

    print("Hit Return Key...")
    time0 = time.time()

    input()

    time1 = time.time()
    print("Execute time ... : %f[s]" %(time1 - time0))
    print()


if __name__ == "__main__":
    main()
